.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,70 @@
+name: "🐛 Bug report"
+description: Report a bug found while using the Berty app.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Have a question? 👉 [Start a new discussion in Discord](https://crpt.fyi/berty-discord)
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+       1. Navigate to the ...
+       2. Click on the user...
+       3. Change name of '...'
+       4. Click Save button...
```

Comment:
```suggestion
       1. Navigate to the ...
       2. Click on the user ...
       3. Change name of ...
       4. Click on the Save button ...
```

Consistency

Reply:
@aeddi , I resolved all issues, please take a look again. 

p.s Thanks for detailed review